Doc/library/sys.rst

```diff
@@ -1524,6 +1524,35 @@ always available.
       This function has been added on a provisional basis (see :pep:`411`
       for details.)  Use it only for debugging purposes.
 
+.. function:: activate_stack_trampoline(backend, /)
+
+   Activate the stack profiler trampoline *backend*.
```

Comment:
Maybe link against th `perf` docs?

Reply:
Yeah, more links :) What are the official perf docs? This, I presume: https://perf.wiki.kernel.org/index.php/Main_Page

Reply:
I sorry, I meant to link against [Doc/howto/perf_profiling.rst](https://github.com/python/cpython/pull/96445/files#diff-3a87b67fbb1a25a5f143ed161778241a9f6285dcc2aa4787c957a2674b6bf70c)

Reply:
We should also link to those in the howto.

Reply:
Added in 31c0eb819c2b951051e5cb12d497db01af28bcd4

Reply:
> Ah sorry, I meant to link against `Doc/howto/perf_profiling.rst`

Aha, that was already in place in the `seealso` just below ;) However, linking to the perf wiki is also nice.